aarch64_qemuvirt/src/main.rs

```diff
@@ -0,0 +1,77 @@
+#![no_std]
+#![no_main]
+// #![feature(custom_test_frameworks)]
+// #![test_runner(crate::kernel_tests::runner)]
+// #![reexport_test_harness_main = "ktests_launch"]
+
+#[cfg(not(target_arch = "aarch64"))]
+compile_error!("Must be compiled as aarch64");
```

Comment:
Is this still necesarry ?

Reply:
Yes, it's a safety check, I think it is better than a cryptic compiler error in the future